src/test/ui/consts/miri_unleashed/const_refers_to_static.stderr

```diff
@@ -1,59 +1,73 @@
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:11:18
+  --> $DIR/const_refers_to_static.rs:14:6
+   |
+LL |     &FOO
+   |      ^^^
+
+warning: skipping const checks
+  --> $DIR/const_refers_to_static.rs:22:18
    |
 LL |     unsafe { &*(&FOO as *const _ as *const usize) }
    |                  ^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:17:5
+  --> $DIR/const_refers_to_static.rs:27:38
+   |
+LL | const REF_EXTERN: &usize = unsafe { &EXTERN };
+   |                                      ^^^^^^
+
+warning: skipping const checks
+  --> $DIR/const_refers_to_static.rs:37:5
    |
 LL |     FOO.fetch_add(1, Ordering::Relaxed)
    |     ^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:17:5
+  --> $DIR/const_refers_to_static.rs:37:5
    |
 LL |     FOO.fetch_add(1, Ordering::Relaxed)
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:24:17
+  --> $DIR/const_refers_to_static.rs:44:17
    |
 LL |     unsafe { *(&FOO as *const _ as *const usize) }
    |                 ^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:29:32
+  --> $DIR/const_refers_to_static.rs:49:32
    |
 LL | const READ_MUT: u32 = unsafe { MUTABLE };
    |                                ^^^^^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:29:32
+  --> $DIR/const_refers_to_static.rs:49:32
    |
 LL | const READ_MUT: u32 = unsafe { MUTABLE };
    |                                ^^^^^^^
 
 warning: skipping const checks
-  --> $DIR/const_refers_to_static.rs:36:6
+  --> $DIR/const_refers_to_static.rs:53:37
    |
-LL |     &FOO
-   |      ^^^
+LL | const READ_EXTERN: usize = unsafe { EXTERN };
+   |                                     ^^^^^^
 
-error[E0080]: it is undefined behavior to use this value
-  --> $DIR/const_refers_to_static.rs:9:1
+warning: any use of this value will cause an error
+  --> $DIR/const_refers_to_static.rs:32:35
    |
-LL | / const REF_INTERIOR_MUT: &usize = {
-LL | |     static FOO: AtomicUsize = AtomicUsize::new(0);
-LL | |     unsafe { &*(&FOO as *const _ as *const usize) }
-LL | |
-LL | | };
-   | |__^ type validation failed: encountered a reference pointing to a static variable
+LL | const DEREF_INTERIOR_MUT: usize = *REF_INTERIOR_MUT;
+   | ----------------------------------^^^^^^^^^^^^^^^^^-
+   |                                   |
+   |                                   constant accesses static
```

Comment:
The error message should be reworded now, something like "constant reading from mutable static".